clients/python/game/models.py

```diff
@@ -24,9 +27,12 @@ def __str__(self):
 Tick: %d
 Map: %s
 Players:
+  %s
+Resources:
   %s""" % (self.tick,
            self.map,
-           "\n  ".join([str(player) for player in self.players])))
+           "\n  ".join([str(player) for player in self.players]),
```

Comment:
That was already there (`Player` PR). Let me show what this does through an example:
```py
>>> " ".join(["my","name","is","Jesse"])
'my name is Jesse'
```
So it adds `" "` between all the elements in the array! It *joins* the array using the given string separator, in other words.

Here, when we do:
```py
[str(player) for player in self.players]
```
We are getting the string representation of each player in an array.

By doing the following:
```py
"\n  ".join(players_as_strings)
```
We are creating an output that looks like (adding new lines and indentation, in other words):
```
  player 1 ...
  player 2 ...
  player 3 ...
  ...
```
Does that make sense? I'll merge because I don't think this is related to this PR, but feel free to ask more questions if it is unclear.